### PR TITLE
Release/v1.11.1

### PR DESCRIPTION
**Pull request by g-iannelli**

### CODE REVIEW

File: docs/releases/v1.11.1.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# OPA Core Module Release 1.11.0
```
Comment (ralgozino) [inline docs/releases/v1.11.1.md:1]:
```suggestion
# OPA Core Module Release 1.11.1
```

File: katalog/kyverno/core/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    newTag: v1.11.0
+  - name: ghcr.io/kyverno/cleanup-controller
+    newName: registry.sighup.io/fury/kyverno/cleanup-controller
+    newTag: v1.11.0
+  - name: ghcr.io/kyverno/reports-controller
+    newName: registry.sighup.io/fury/kyverno/reports-controller
+    newTag: v1.11.0
+  - name: bitnami/kubectl
+    newName: registry.sighup.io/fury/bitnami/kubectl
+    newTag: "1.26.10"
```
Comment (ralgozino) [inline katalog/kyverno/core/kustomization.yaml:34]: nitpick: missing newline
```suggestion
    newTag: "1.26.10"

```

File: katalog/kyverno/kustomization.yaml [hunk 3]
```diff
@@ -6,30 +6,6 @@
...
-    newTag: v1.11.0
-  - name: ghcr.io/kyverno/reports-controller
-    newName: registry.sighup.io/fury/kyverno/reports-controller
-    newTag: v1.11.0
-  - name: bitnami/kubectl
-    newName: registry.sighup.io/fury/bitnami/kubectl
-    newTag: "1.26.10"
\ No newline at end of file
+  - core
+  - policies
```
Comment (ralgozino) [inline katalog/kyverno/kustomization.yaml:11]: nitpick: missing new line
```suggestion
  - policies

```

**Review by ralgozino — CHANGES_REQUESTED** [review]

File: katalog/kyverno/core/kustomization.yaml (hunk 2 above)
Comment (g-iannelli) [reply katalog/kyverno/core/kustomization.yaml:34]: Maybe it's good also to upgrade kubectl to 1.27.10

**Review by nutellinoit — APPROVED** [review]
LGTM

**Pull request merged**